includes/class-freemius.php

```diff
@@ -918,9 +960,54 @@ private function _find_caller_plugin_file( $is_init = false ) {
 				'path' => $id_slug_type_path_map[ $this->_module_id ]['path'],
 			);
 
-			return $id_slug_type_path_map[ $this->_module_id ]['path'];
+			return $this->get_absolute_path( $id_slug_type_path_map[ $this->_module_id ]['path'] );
 		}
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 1.2.3
+         *
+         * @param string $path
+         *
+         * @return string
+         */
+        private function get_relative_path( $path ) {
+            $module_root_dir = fs_normalize_path( trailingslashit( $this->is_plugin() ?
+                WP_PLUGIN_DIR :
+                get_theme_root() ) );
+
+            if ( 0 === strpos( $path, $module_root_dir ) ) {
+                $path = str_replace( $module_root_dir, '', $path );
+            }
+
+            return $path;
+        }
+
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 1.2.3
+         *
+         * @param string      $path
+         * @param string|bool $module_type
+         *
+         * @return string
+         */
+        private function get_absolute_path( $path, $module_type = false ) {
+            $is_plugin = empty( $module_type ) ?
+                $this->is_plugin() :
+                ( WP_FS__MODULE_TYPE_PLUGIN === $module_type );
+
+            $module_root_dir = fs_normalize_path( $is_plugin ?

```

Comment:
@fajardoleo I suggest creating a helper method to calculate the `get_module_root_dir_path( $module_type = false )` since you have it here and in `get_relative_path()`.